controller/lib/core/sensors.cpp

```diff
@@ -123,22 +123,23 @@ Pressure Sensors::ReadPressureSensor(Sensor s) {
 }
 
 /*static*/ VolumetricFlow Sensors::PressureDeltaToFlow(Pressure delta) {
-  // TODO(jlebar): Make these constexpr once we have a C++ standard library
-  // PortArea must be larger than the ChokeArea [meters^2]
-  float venturiPortArea = diameter_to_area_m2(DEFAULT_VENTURI_PORT_DIAM);
-  float venturiChokeArea = diameter_to_area_m2(DEFAULT_VENTURI_CHOKE_DIAM);
+  constexpr float venturiPortArea =
+      diameter_to_area_m2(DEFAULT_VENTURI_PORT_DIAM);
+  constexpr float venturiChokeArea =
+      diameter_to_area_m2(DEFAULT_VENTURI_CHOKE_DIAM);
   //[meters^4]
-  float venturiAreaProduct = venturiPortArea * venturiChokeArea;
+  constexpr float venturiAreaProduct = venturiPortArea * venturiChokeArea;
   // Equivalent to 1/sqrt(A1^2 - A2^2) guaranteed never to have a negative
   // radicand [1/meters^2]
-  float bernoulliAreaDivisor =
-      1.0f / sqrtf(venturiPortArea * venturiPortArea -
-                   venturiChokeArea * venturiChokeArea);
+  const float bernoulliAreaDivisor =
+      1.0f / std::sqrt(venturiPortArea * venturiPortArea -
+                       venturiChokeArea * venturiChokeArea);
 
-  float sgn = copysignf(1.0f, delta.kPa());
-  return cubic_m_per_sec(sgn * sqrtf(2 / DENSITY_OF_AIR_KG_PER_CUBIC_METER) *
+  float sgn = std::copysignf(1.0f, delta.kPa());
```

Comment:
nit: instead of multiplying by sgn, could instead copysignf() into any of the existing multiplicands below.

Reply:
Did that, but then realized I wanted to make a bigger change otherwise it looked weird to me.  Would you mind having another look?